src/target_detector/TargetDetector.cpp

```diff
@@ -111,8 +115,8 @@ void gazepoint_callback(const clm_ros_wrapper::GazePointAndDirection::ConstPtr&
             }
 
             //inside/outside check for the screen
-            if ((-1)* GAZE_ERROR > gaze_point_wf.getZ() || screenHeight * sin(screenAngle)  + GAZE_ERROR < gaze_point_wf.getZ()
-                || gaze_point_wf.getX() > screenWidth / 2 + GAZE_ERROR || gaze_point_wf.getX() < (-1) * screenWidth / 2 - GAZE_ERROR)
+            if ((-2) * GAZE_ERROR > gaze_point_wf.getZ() || screenHeight * sin(screenAngle)  + GAZE_ERROR < gaze_point_wf.getZ()
```

Comment:
Please document what you are doing here, otherwise after you'll leave nobody will be able to understand what is going on!